source/OptimaJet.DWKit.StarterApplication/Controllers/OrganizationInvitesController.cs

```diff
@@ -1,4 +1,3 @@
-﻿using System;
```

Comment:
I left the default "using System;" in there when I went through and removed unused namespaces.  I figured that it could be common and it might be weird to type a classname you expect to just be there (during SDD) and not be.